python/social_bot/envs/grocery_ground.py

```diff
@@ -216,7 +222,7 @@ def run(self, agent, world):
         x = self._agent_init_pos[0] + random.random() * self._random_range
         y = self._agent_init_pos[1] + random.random() * self._random_range
         orient = (random.random() - 0.5) * np.pi
-        agent.set_pose(np.array([x, y, 0.6]), np.array([0, 0, orient]))
+        agent.set_pose((np.array([x, y, 0.6]), np.array([0, 0, orient])))
```

Comment:
How can this run successfully before? It does not match the function signature.

Reply:
Before merge last PR I removed the logic of randomly setting pose towards the target, so left a bug here. Now we have a unit test for it hopefully this won't happen again.